README.md

```diff
@@ -33,10 +33,10 @@ The API reference is located at [realm.io/docs/javscript/latest/api](https://rea
 In case you don't want to use the precompiled version on npm, you can build Realm yourself from source. You’ll need an Internet connection the first time you build in order to download the core library.
 
 Prerequisites:
-- Node 4.0+
+- Node 6.5.0
```

Comment:
Is there a need to see if the test cases pass with each supported major version before I update the docs and resend the PR?